requirements/test.txt

```diff
@@ -10,6 +10,5 @@ isort==4.2.15
 mccabe==0.5.2
 mock==2.0.0
 model_mommy==1.2.6
-moto==0.4.30
+moto==0.4.27
```

Comment:
I wonder if there's a good reason to change `moto` for a nearly four year old version of Jinja2, instead of updating Jinja2 to something more recent.

Reply:
@willbarton I think @chosak had some reservations about upgrading Jinja2, but he can say.

Reply:
@acrewdson @willbarton no real reservations per se, just that an upgrade might require testing/verifying the changes in our Jinja templates vs. downgrading moto which would require no extra work at this time.